xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -110,4 +114,5 @@ class CProcessInfo
   // player states
   CCriticalSection m_stateSection;
   bool m_stateSeeking;
+  std::atomic_int m_levelVQ;
```

Comment:
everything not in the top level folder of VideoPlayer is not supposed to be used by other components. CProcessInfo represents the state and behavior of VP that is totally different to the one of RP